lib/src/install.rs

```diff
@@ -627,9 +629,10 @@ pub(crate) struct RootSetup {
     device: Utf8PathBuf,
     rootfs: Utf8PathBuf,
     rootfs_fd: Dir,
+    rootfs_uuid: Option<String>,
     /// If true, do not try to remount the root read-only and flush the journal, etc.
     skip_finalize: bool,
-    boot: MountSpec,
+    boot: Option<MountSpec>,
```

Comment:
Tangentially related...it's in *precisely* this kind of code refactoring that Rust is great.  The compiler found all the places I needed to change.

Reply:
That is awesome indeed!